examples/message/labview/LabVIEW.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using NationalInstruments.SystemLink.Clients.Message;
+
+namespace NationalInstruments.SystemLink.Clients.Examples.Message
+{
+    /// <summary>
+    /// Example for the SystemLink Message client API that communicates with
+    /// Host.vi from the Async Messaging LabVIEW example.
```

Comment:
Hmm... maybe add "the" in front of "Host.vi" here?

Reply:
Yeah, I've always found it awkward to refer to VIs. I usually select either "the Host VI" or "Host.vi". But I can change it to "the Host.vi" if you think it makes more sense that way.

Reply:
I agree. It's always awkward referring to VIs. Let's keep it as is.